sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -1362,11 +1362,27 @@ private void restorePseudonymizedDto(ContactDto dto, Contact existingContact, Co
 		if (existingContactDto != null) {
 			boolean isInJurisdiction = contactJurisdictionChecker.isInJurisdictionOrOwned(existingContact);
 			User currentUser = userService.getCurrentUser();
+
 			Pseudonymizer pseudonymizer = Pseudonymizer.getDefault(userService::hasRight);
 
+			String followUpComment = null;
+			if (dto.isPseudonymized() || !pseudonymizer.isAccessible(ContactDto.class, "followUpComment", isInJurisdiction)) {
```

Comment:
Please use `CaseDataDto.FOLLOWUP_COMMENT` instead of hardcoded `"followUpComment"`